iib/web/config.py

```diff
@@ -20,6 +20,7 @@ class Config(object):
     IIB_ADDITIONAL_LOGGERS: List[str] = []
     IIB_AWS_S3_BUCKET_NAME: Optional[str] = None
     IIB_BINARY_IMAGE_CONFIG: Dict[str, Dict[str, str]] = {}
+    IIB_GRAPH_MODE_ALLOW_LIST: List[str] = []
```

Comment:
This is a list of allowed Index Image pull specs on which the graph mode can be used. It will be set in iib-playbooks. Here we are simply defalting it to an empty list